etc/docker.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+# script to aid in local testing of linux platforms
```

Comment:
Happy to remove, but this is a handy script to run our tests locally in linux containers.